src/rdb.c

```diff
@@ -219,3 +222,51 @@ void series_rdb_save(RedisModuleIO *io, void *value) {
     }
     RedisModule_DictIteratorStop(iter);
 }
+
+int aux_load_count = 0;
+
+int auxLoad(RedisModuleIO *io, int encver, int when) {
+    if (REDISMODULE_AUX_BEFORE_RDB) {
+        replicaof_in_progress = true;
+        aux_load_count++;
+
+        // replicate auxLoad (fixes a bug in Redis < 6.2.10 auxLoad isn't replicated)
+        RedisModuleCallReply *reply =
```

Comment:
Why not just use RM_Replicate? We say that this is what we need to check, if it is OK to call it here...